src/batGc.mliv

```diff
@@ -182,7 +182,8 @@ external counters : unit -> float * float * float = "caml_gc_counters"
     is as fast at [quick_stat]. *)
 
 ##V>=4.4##external minor_words : unit -> (float [@unboxed])
-##V>=4.4##  = "caml_gc_minor_words" "caml_gc_minor_words_unboxed" [@@noalloc]
+##V>=4.4##  = "caml_gc_minor_words" "caml_gc_minor_words_unboxed"
+##V<4.5##     [@@noalloc]
```

Comment:
I'm afraid this will include "[@@noalloc]" on 4.03, which would be wrong. I think you need to use `##V=4.4##` instead.